common/djangoapps/nodebb/views.py

```diff
@@ -59,7 +59,8 @@ def nodebb_forum_discussion(request, course_id):
         "is_community_topic_link": is_community_topic_link,
         "browse_teams_link": browse_teams_link,
         "course_has_ended": current_course.has_ended(),
-        "courses_page_link": reverse("courses")
+        "courses_page_link": reverse("courses"),
+        "room_id": course_community.community_url.split('/')[0]
```

Comment:
can this be a method that returns room_id and handle any errors that could occur in that function?

Reply:
I don't think any errors would occur here. And since this logic is only being used once, implementing a separate function for it just seems to complicate the logic. Please correct me if I am wrong